modules/python/test/test_misc.py

```diff
@@ -607,6 +608,21 @@ def test_result_rotated_rect_issue_20930(self):
         self.assertTrue(isinstance(rr, tuple), msg=type(rrv))
         self.assertEqual(len(rr), 3)
 
+    def test_nested_function_availability(self):
+        self.assertTrue(hasattr(cv.utils, "nested"),
+                        msg="Module is not generated for nested namespace")
+        self.assertTrue(hasattr(cv.utils.nested, "testEchoBooleanFunction"),
+                        msg="Function in nested module is not available")
+        self.assertEqual(sys.getrefcount(cv.utils.nested), 3,
```

Comment:
In contrast with 3.4 branch - reference count should be 3, due to changes in how modules are exported.